osu.Game/Storyboards/Drawables/DrawableStoryboard.cs

```diff
@@ -57,7 +57,13 @@ public bool Passing
         public DrawableStoryboard(Storyboard storyboard)
         {
             Storyboard = storyboard;
+
             Size = new Vector2(640, 480);
+
+            bool onlyHasVideoElements = !Storyboard.Layers.Any(l => l.Elements.Any(e => !(e is StoryboardVideo)));
```

Comment:
Can also be written as

```suggestion
            bool onlyHasVideoElements = !Storyboard.Layers.Where(l => !(l is StoryboardLayerVideo)).Any(l => l.Elements.Any());
```

... but I *think* what I have is easier to mentally parse.

Reply:
Or, `Storyboard.Layers.SelectMany(l => l.Elements).Any(e => !(e is StoryboardVideo))`?

Reply:
Sure, that works as well. Not sure which is more efficient if any, but I guess it doesn't matter too much in this usage.